cookbook/bundles/extension.rst

```diff
@@ -125,3 +125,43 @@ Using Configuration to Change the Services
 The Extension is also the class that handles the configuration for that
 particular bundle (e.g. the configuration in ``app/config/config.yml``). To
 read more about it, see the ":doc:`/cookbook/bundles/configuration`" article.
+
+Adding Classes to Compile
+-------------------------
+
+In order to make applications run as fast as possible on production environment,
+Symfony creates a big ``classes.php`` file in the cache directory. This file
+aggregates the contents of the PHP classes that are used in every request,
+reducing the I/O operations related to those classes.
+
+Your own bundles can add classes into this file thanks to the ``addClassesToCompile()``
+method. Define the classes to compile as an array of their fully qualified class
+names::
+
+    // ...
+    public function load(array $configs, ContainerBuilder $container)
+    {
+        // ...
+
+         $this->addClassesToCompile(array(
+            'AppBundle\\Manager\\UserManager',
+            'AppBundle\\Service\\Slugger',
+            // ...
+        ));
+    }
+
+.. note::
+
+    If some class extends from other classes, all its parents are included
+    automatically in the list of classes to compile.
+
+After adding to compile all the classes commonly used by your bundle, you can
+expect a minor performance improvement.
```

Comment:
But then we explain how to do something ... but not why it's useful. But I'm going to reword it.